Gulpfile.ts

```diff
@@ -156,14 +156,16 @@ const librarySourceMap = [
     { target: "lib.scripthost.d.ts", sources: ["header.d.ts", "scripthost.d.ts"] },
 
     // JavaScript library
+    { target: "lib.es3.d.ts", sources: ["header.d.ts", "es3.d.ts"] },
     { target: "lib.es5.d.ts", sources: ["header.d.ts", "es5.d.ts"] },
     { target: "lib.es2015.d.ts", sources: ["header.d.ts", "es2015.d.ts"] },
     { target: "lib.es2016.d.ts", sources: ["header.d.ts", "es2016.d.ts"] },
     { target: "lib.es2017.d.ts", sources: ["header.d.ts", "es2017.d.ts"] },
     { target: "lib.esnext.d.ts", sources: ["header.d.ts", "esnext.d.ts"] },
 
     // JavaScript + all host library
-    { target: "lib.d.ts", sources: ["header.d.ts", "es5.d.ts"].concat(hostsLibrarySources) },
+    { target: "lib.d.ts", sources: ["header.d.ts", "es3.d.ts"].concat(hostsLibrarySources) },
+    { target: "lib.es5.full.d.ts", sources: ["header.d.ts", "es5.d.ts"].concat(hostsLibrarySources) },

```

Comment:
i would keep lib.d.ts to be es5 and create `lib.es3.full` just to avoid breaking users how depend on a specific meaning of `lib.d.ts.`